assets/shaders/circle_shader.wgsl

```diff
@@ -0,0 +1,48 @@
+// This shader draws a circular progress bar
+#import bevy_render::view View
+
+struct VertexOutput {
```

Comment:
Maybe move the VertexOutput into an import to make less boilerplate?

Reply:
I think this is the right approach. Should I just add the export at the top of the current shader, or should I create a new file that only houses the struct itself that has the export?

Reply:
@MarkusTheOrt Yep, that would be nice!

Reply:
I added the output inside the `material.wgsl` shader and have also implemented it in the `circle_shader.wgsl` 👍

Reply:
@MarkusTheOrt Oh, sorry, I misread your comment. I think that a new file `ui_vertex_output.wgsl` that contains only `UiVertexOutput` would be better, like it's done in https://github.com/bevyengine/bevy/blob/main/crates/bevy_sprite/src/mesh2d/mesh2d_vertex_output.wgsl

Reply:
Thank you! :)